modules/tensor_mechanics/doc/content/documentation/systems/BCs/CoupledPressureBC.md

```diff
@@ -0,0 +1,24 @@
+# Coupled Pressure BC
+
+## Description
+
+The boundary condition `CoupledPressureBC` applies a force computed in a variable to a mesh boundary.
+A `component` of the normal vector to the mesh surface (0, 1, or 2 corresponding to the $\hat{x}$, $\hat{y}$, and $\hat{z}$ vector components) is used to determine the direction in which to apply the traction.
+
+!alert note
+The boundary condition is always applied to the displaced mesh.
+
+The `CoupledPressureBC` is typically used in a multi-app scenario.
+The pressure variable can be computed by a sub-app (it can be for example a flow code) and then transferred into an auxiliary variable, which is then coupled into this boundary condition so that it is applied in the master app.
+
+A set of `Pressure` boundary conditions applied to multiple variables in multiple components can be defined with the [PressureAction](/BCs/Pressure/PressureAction.md).
```

Comment:
Did you mean 'A set of \`CoupledPressureBC\` boundary conditions`?

Reply:
To me, `Pressure` here indicated type of the BC, rather then C++ class name, but if you think it should read `CoupledPressureBC`, I can change that.

Reply:
I would prefer it be CoupledPressureBC.